nemo/core/classes/exportable.py

```diff
@@ -186,13 +218,14 @@ def export(
                             onnx_model = gs.export_onnx(graph.fold_constants().cleanup())
                             onnx.checker.check_model(onnx_model, full_check=True)
                             onnx.save(onnx_model, output)
-
-                    return onnx_model
+                    exported = onnx_model
                 else:
                     raise ValueError(f'Encountered unknown export format {format}.')
         finally:
             typecheck.set_typecheck_enabled(enabled=True)
-        return [output]  # Subclasses may create more than one file.
+            if forward_method:
+                type(self).forward = old_forward_method
+        return exported
```

Comment:
@borisfom in EFF I am expecting the export method to return two lists:
 - filenames (containing list of onnx filenames)
 - descriptions (containing list of descriptions what actually is there)

Please analyse e.g. Punctuation and Capitalization Model:
https://github.com/NVIDIA/NeMo/blob/r1.0.0b4/nemo/collections/nlp/models/token_classification/punctuation_capitalization_model.py#L566


This is how I use this information in EFF:
https://gitlab-master.nvidia.com/tlt/eff/-/blob/main/eff/cookbooks/onnx_cookbook.py#L178 

Please update the information you return here accordingly. Thx